client/lib/importer/actions.js

```diff
@@ -129,14 +129,12 @@ export function finishUpload( importerId, importerStatus ) {
 	} );
 }
 
-export function mapAuthor( importerId, sourceAuthor, targetAuthor ) {
-	Dispatcher.handleViewAction( {
-		type: IMPORTS_AUTHORS_SET_MAPPING,
-		importerId,
-		sourceAuthor,
-		targetAuthor
-	} );
-}
+export const mapAuthor = ( importerId, sourceAuthor, targetAuthor ) => ( {
```

Comment:
There is always a debate whether to test or not so simple functions. Personally when I write code from scratch then I cover them with tests, because it's super easy and fast. I also tend to skip so simple tests when I refactor existing untested code ;)